CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 ## v0.18.0 [04-13-2017]
 
+v0.18.0 contains our latest work to support both Polymer 1.x & 2.0 projects. There are a bunch of big new features included in this update, as well as several breaking changes since the latest version. Here is a quick summary of the major changes for anyone who is updating from our previous `latest`/`v0.17.0` version:
+
+- **New Polymer 2.0 Templates**: `polymer init` has added new Polymer 2.0 templates for starter elements, applications, and our latest Polymer Starter Kit & Shop applications. Run `polymer init` to see the whole list.
+- **Updated `lint` Command**: `polymer lint` is now powered by our newest version of [polymer-linter](https://github.com/Polymer/polymer-linter), which provides even more control over different lint rules. Run `polymer help lint` for more information.
```

Comment:
For the linter I think the big selling points are:
 - it shows you your code with the problem underlined
 - you can configure which rules to run

so something like:

> which can show you the exact location of the warning in your code, and is much more configurable. See `polymer help lint` for more information.